schema/context.go

```diff
@@ -0,0 +1,37 @@
+package schema
+
+import (
+	"context"
+)
+
+type bodyExtCtxKey struct{}
+
+type bodyActiveCountCtxKey struct{}
+
+func WithActiveCount(ctx context.Context) context.Context {
+	return context.WithValue(ctx, bodyActiveCountCtxKey{}, true)
+}
+
+func ActiveCountFromContext(ctx context.Context) bool {
```

Comment:
This function isn't used anywhere. Either we missed adding it somewhere or the refactoring made most of the context related checks obsolete.